script/update-doc-repo.pl

```diff
@@ -966,7 +983,7 @@ sub process_docbook_docdir {
 		do_debug("- Processing $name");
 
 		my $mappedname = (exists $mapping->{$name}? $mapping->{$name} : $name);
-		my $to = File::Spec->catdir($INSTALLDIR, $mappedname);
+		my $to = File::Spec->catdir($INSTALL$DIR_HANDLE, $mappedname);
```

Comment:
I think this was mistakenly changed.

Reply:
I went ahead and pushed the fix for this in 09221ba03061f827f11745fc76aa389094aefa11.